packages/example-app-base/README.md

```diff
@@ -1,12 +1,12 @@
 # @uifabric/example-app-base
 
-Components and utilities used to build documentation sites for various [Office UI Fabric React](https://dev.microsoft.com/fabric) packages.
+**Deprecated** components and utilities used to build internal documentation sites and inner loops for various [Fluent UI React](https://developer.microsoft.com/en-us/fluentui) ([formerly Office UI Fabric React](https://developer.microsoft.com/en-us/office/blogs/ui-fabric-is-evolving-into-fluent-ui/)) packages.
```

Comment:
Why is this marked as Deprecated?

Reply:
Because we really really don't want people using this going forward. They should use storybook for their demo apps.